cmd/bucket-handlers.go

```diff
@@ -1809,6 +1809,10 @@ func (api objectAPIHandlers) PutBucketObjectLockConfigHandler(w http.ResponseWri
 		return
 	}
 
+	// Audit log tags.
```

Comment:
Which `mc` command covers this path?
`mc retention set governance` doesn't follow this rather it goes through `PutObjectRetention`

Reply:
@shtripat Interesting.. I have been testing with `mc retention set --default governance 1d local/lockedbucket` and it is audit logging as follows:

`...."api":{"name":"PutBucketObjectLockConfig","bucket":"lockedbucket","status":"OK","statusCode":20...`

Reply:
I will take a look at the PutObjectRetention handler too and see if I can trigger that

Reply:
Ok I can add handling for PutObjectRetention, but just note that it's not specifically what the customer was asking for. Object retention tag will look like:

`"tags":{"retention":"Mode: GOVERNANCE, RetainUntilDate: 2024-12-15 01:24:26 +0000 UTC"`

Reply:
> @shtripat Interesting.. I have been testing with `mc retention set --default governance 1d local/lockedbucket` and it is audit logging as follows:
> 
> `...."api":{"name":"PutBucketObjectLockConfig","bucket":"lockedbucket","status":"OK","statusCode":20...`

ok, let me try this